src/case_gather/views.py

```diff
@@ -11,7 +11,13 @@
 @api_view(['GET', 'POST'])
 @permission_classes((IsAuthenticatedOrReadOnly,))
 def case_list(request):
+    '''
+    get:
+        Get a list of all cases.
 
+    post:
```

Comment:
If that's the case, then `@api_view(['GET', 'POST'])` should be changed to `@api_view(['GET''])`since Swagger is autogenerated and will just create everything that has been defined.

Or we can add `NOT_IMPLEMENTED` as the docstring comment.